test/app.hooks.test.js

```diff
@@ -279,3 +282,87 @@ describe('removeObjectIdInData hook', () => {
 		expect(_id.toString()).to.not.equal(response._id.toString());
 	});
 });
+
+describe('handleAutoLogout hook', () => {
+	let fut;
+	let redisHelper;
+
+	before(async () => {
+		mockery.enable({
+			warnOnReplace: false,
+			warnOnUnregistered: false,
+			useCleanCache: true,
+		});
+		mockery.registerMock('redis', redisMock);
+
+		delete require.cache[require.resolve('../src/utils/redis')];
+		/* eslint-disable global-require */
+		redisHelper = require('../src/utils/redis');
+		fut = require('../src/app.hooks').handleAutoLogout;
```

Comment:
handleAutoLogout lädt kein redis, muss das hier required werden, oder reicht auch oben als normaler const myFunktion = require ?

Reply:
handleAutoLogaut lädt die redisHelper (der , und die wiederum lädt redis. in dem Moment wollen wir dass der Mock anstatt einer echten redis connection geladen wird.
Und initialisieren wollen wir auch nur die neue instanz der redishelper...